app/utils/actor/AuthTokenCleanerActor.scala

```diff
@@ -0,0 +1,82 @@
+package utils.actor
+
+import java.text.SimpleDateFormat
+import java.sql.Timestamp
+import java.util.{Calendar, Locale, TimeZone}
+import java.time.Instant
+import javax.inject.Inject
+
+import akka.actor.{Actor, Cancellable, Props}
+import models.services.{AuthTokenService, AuthTokenServiceImpl}
+import models.daos.slick.AuthTokenDAOSlick
+import play.api.Logger
+
+import scala.concurrent.duration._
+import play.api.libs.concurrent.Execution.Implicits._
+
+// Template code comes from this helpful StackOverflow post:
+// https://stackoverflow.com/questions/48977612/how-to-schedule-complex-tasks-using-scala-play-2-3/48977937?noredirect=1#comment84961371_48977937
+class AuthTokenCleanerActor extends Actor {
+
+  private var cancellable: Option[Cancellable] = None
+  val TIMEZONE = TimeZone.getTimeZone("UTC")
+  val authTokenService : AuthTokenService = new AuthTokenServiceImpl(new AuthTokenDAOSlick())
+
+  override def preStart(): Unit = {
+    super.preStart()
+
+    // If we want to update the user_stat table at 2:00am every day, we need to figure out how much time there
```

Comment:
Can you change this to not say "user_stat table"

Reply:
Got it